src/elements/content-uploader/progressCellRenderer.js

```diff
@@ -58,7 +61,7 @@ const getErrorMessage = (errorCode: ?string, itemName: ?string) => {
     }
 };
 
-export default () => ({ rowData }: Props) => {
+export default (shouldShowUpgradeCTAMessage?: ?boolean) => ({ rowData }: Props) => {
```

Comment:
Is this saying that the prop is optional and it also that it's not always a boolean?